framework/src/org/checkerframework/qualframework/base/QualifiedTypeFactoryAdapter.java

```diff
@@ -65,10 +65,8 @@ void doPostInit() {
     }
 
     @Override
-    protected QualifierDefaults createQualifierDefaults() {
-        QualifierDefaults result = super.createQualifierDefaults();
-        getCheckerAdapter().setupDefaults(result);
-        return result;
+    protected void addCheckedCodeDefaults(QualifierDefaults defs) {
+        getCheckerAdapter().setupDefaults(defs);
```

Comment:
Recommend adding a short note indicating why super.addCheckedCodeDefaults is not called. (I am not familiar with this class - perhaps this would be clear to someone who is.)